packages/page-accounts/src/modals/Import.tsx

```diff
@@ -5,7 +5,7 @@ import type { KeyringPair, KeyringPair$Json } from '@polkadot/keyring/types';
 import type { ActionStatus } from '@polkadot/react-components/Status/types';
 import type { ModalProps } from '../types';
 
-import React, { useCallback, useMemo, useState } from 'react';
+import React, { Dispatch, SetStateAction, useCallback, useMemo, useState } from 'react';
```

Comment:
```suggestion
import React, { useCallback, useMemo, useState } from 'react';
```

And then do an explict type-only import at the top -

```js
import type { Dispatch, SetStateAction } from 'react';
```